### DISCUSSION TITLE
Connected fields using the form builder

### DISCUSSION THREAD

**Discussion by Tofandel** [area17/twill#2552] [Q&A] [answered]
It seems it's the only missing twill component in the form builder, we should definitely add the ConnectedField as a form service, similar to `Columns`

**Comment by ifox** [comment, marked as answer]
This is already available, see https://github.com/area17/twill/pull/2323

**Reply by Tofandel** [reply]
Ah yes thanks, we ought to update the documentation on connected fields then
